server/mediaserver.go

```diff
@@ -393,20 +387,61 @@ func (s *LivepeerServer) registerConnection(rtmpStrm stream.RTMPVideoStream) (*r
 		profile:     &vProfile,
 		params:      params,
 		sessManager: NewSessionManager(s.LivepeerNode, params, playlist),
+		lastUsed:    time.Now(),
 	}
+
 	s.connectionLock.Lock()
 	s.rtmpConnections[mid] = cxn
 	s.lastManifestID = mid
 	s.lastHLSStreamID = hlsStrmID
 	sessionsNumber := len(s.rtmpConnections)
 	s.connectionLock.Unlock()
+
+	ticker := getConnectionTicker()
+	go func(s *LivepeerServer, mid core.ManifestID) {
+		for _ = range ticker.C {
+			lastUsed := s.rtmpConnections[mid].lastUsed
+			now := time.Now()
+			diff := now.Sub(lastUsed)
+			if diff.Hours() >= 1 || diff.Minutes() >= 1 {
```

Comment:
One hour timeout is a really long time, given that segments are just a few seconds long. One minute seems enough.

Assuming we reuse `connectionRefreshInterval` as the 1-minute timeout, this approach is more terse:

```golang
if time.Since(lastUsed) > connectionRefreshInterval {
 // ... remove RTMP stream ...
}
```

Reply:
True, a minute timeout would negate the need to check for an hr difference